transformations/transformer_replace/test.json

```diff
@@ -0,0 +1,46 @@
+{
+  "type": "transformer_replace",
+  "test_cases": [{
+      "class": "TransformerReplace",
+      "inputs": {
+        "sentence": "Andrew finally returned the French book to Chris that I bought last week"
+      },
+      "outputs": [{
+        "sentence": "I finally returned the French book to Chris Evans I wrote last week"
```

Comment:
Looks like the only sentence changed is this one? All the other outputs are just the same as the inputs? I would suggest at least filtering out un-perturbed sentences?